.devcontainer/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# [Choice] Debian version: buster, stretch
```

Comment:
yeah I don't think this belongs in git ;)

Reply:
agreed... I need it for my weird setup at the moment. I promise to remove it before merge though 🤞